gradoop-flink/src/main/java/org/gradoop/flink/model/impl/operators/fusion/functions/LeftElementId.java

```diff
@@ -0,0 +1,36 @@
+/*
+ * This file is part of Gradoop.
+ *
+ * Gradoop is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * Gradoop is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with Gradoop. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package org.gradoop.flink.model.impl.operators.fusion.functions;
+
+import org.apache.flink.api.java.functions.KeySelector;
+import org.apache.flink.api.java.tuple.Tuple2;
+import org.gradoop.common.model.api.entities.EPGMElement;
+import org.gradoop.common.model.impl.id.GradoopId;
+
+/**
+ * Left projection with id obtained.
+ * @param <K> Element where the id is extracted
+ *
+ */
+public class LeftElementId<K extends EPGMElement> implements KeySelector<Tuple2<K, GradoopId>,
```

Comment:
implements -> new line

try forwarding "f0.id" and see if this works in the test